resources/views/dashboard/widgets/forms/_preview.blade.php

```diff
@@ -45,18 +45,8 @@
 
             <x-grid>
                 <div>
```

Comment:
Why do you keep the div around the include in this form, but not in the other form? Seems like it's useless here